hooks/useFcm.ts

```
@@ -95,12 +95,14 @@ export default function useFcm(
           // 방장이 방 참여 요청을 수락한 경우 : 사용자가 쿼리 무효화
           case 'ACCEPT_ROOM_JOIN':
             const response = await checkHasRoom();
-            setRoomInfo(response.result);
+            setRoom(response.result);
 
             await queryClient.invalidateQueries({ queryKey: [`/rooms/exist`] });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)